reconciletags/reconciletags_main.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Reads json files mapping docker digests to tags and reconciles them.
+"""
+
+import argparse
+import json
+import logging
+import os
+import shutil
+import unittest
+from containerregistry.tools import patched
+from reconciletags import tag_reconciler
+from reconciletags import data_integrity_test
+from reconciletags import config_integrity_test
+
+
+def create_temp_dir(files):
+    dir = "../config/tag/"
+    os.makedirs(dir)
+
+    for file in files:
+        if os.path.isfile(file):
+            shutil.copy(file, dir)
+        else:
+            raise AssertionError("{0} is not a valid file".format(file))
+
+
+def delete_temp_dir():
```

Comment:
This scares me a bit, can we make whatever is creating this use a real tempdir instead of a path in the repo?